### ISSUE TITLE
lumo

### ISSUE THREAD

**Issue by borkdude** [clj-commons/wishlist#4]
https://github.com/anmonteiro/lumo

**Comment by danielcompton** [comment]
Yeah, for CLJ Commons to adopt Lumo, I'd like to see someone (or maybe a few people) volunteer who can commit to maintaining it for some time, ideally with at least one person who has contributed to Lumo before.

**Comment by iku000888** [comment]
I think we should acknowledge that maintaining lumo is going to be a lot of work, so I would like to throwout some clojuriststogether ideas.

## Idea 1

A clojuriststogether project that funds a past lumo contributor.

## Idea 2

Pay Antonio for a series of recorded group(future volunteers) coding sessions working on lumo, ideally showing how a newbie would familiarize to the codebase along with prerequisite knowledge and workflow tips.

I think something like this above would be really useful, because I think that the barrier to the first patch is extremely high for lumo; I tackled the lumo codebase recently and gave up after not being able to get the build working for about an hour. (Places that I got stuck were boot custom tasks/npm realm stuff/building the node runtime stuff)

If there is some positive feedback (and of course Antonio's consent), I would be more than happy to submit a proposal.

**Comment by arichiardi** [comment]
I can only share the sentiment of it being quite some amount of work and I had at some point different ideas on how to get rid of at least the embedding of the whole node runtime...however I never got close to POC that.
The moving parts are many, not that it wouldn't be doable and it is a stretch as one person project IMHO but I would love for it to stay alive.

**Comment by MalloZup** [comment]
I personally would vote to close because the issue is stale more or less then 1 year. If a motivated maintainer comes from shadow it can reopen the issue :bat:
